src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -716,10 +719,14 @@
                                         Grid.Row="0"
                                         Style="{StaticResource MenuExpanderStyle}" 
                                         IsExpanded="{Binding PreferencesTabs[Features].ExpanderActive, Converter={StaticResource ExpandersBindingConverter}, ConverterParameter=PackagePathsExpander}"
-                                        Header="{x:Static p:Resources.PackagePathsExpanderName}">
-                                <StackPanel Orientation="Vertical" Margin="0,6,0,0">
-                                    <Label>SOME CONTENT</Label>
-                                </StackPanel>
+                                
+                                        Header ="{x:Static p:Resources.PackagePathsExpanderName}">
+                             
+                                <ScrollViewer Height="250" VerticalScrollBarVisibility="Auto">
```

Comment:
@sm6srw something like this might fix our scrolling problem

Reply:
Yes, I got it working by specifying a height.